cloudmanager/automation_config.go

```diff
@@ -181,11 +181,11 @@ type SystemLog struct {
 
 // Args26 part of the internal Process struct
 type Args26 struct {
-	NET         Net         `json:"net,omitempty"`
-	Replication Replication `json:"replication,omitempty"`
-	Sharding    Sharding    `json:"sharding,omitempty"`
-	Storage     Storage     `json:"storage,omitempty"`
-	SystemLog   SystemLog   `json:"systemLog,omitempty"`
+	NET         Net          `json:"net"`
+	Replication *Replication `json:"replication,omitempty"`
```

Comment:
`Replication` and `Sharding` are mutually exclusive if one is set the other can be present (even blank) so this needs to be a pointer so the serializer ignores nil

Reply:
These comments are super useful - consider adding them directly in the code as it will help others when we split this into a separate library!

Reply:
done